scripts/update_package_versions.py

```diff
@@ -712,54 +588,298 @@ def bump_version_in_yaml(
     loader.dump(config, open(configuration_file_path, "w"))
 
 
-def process_package(package_id: PackageId, is_ambiguous: bool) -> None:
-    """
-    Process a package.
+class Updater:
+    """PAckage versions updter tool."""
 
-    - check version in registry
-    - make sure, version is exactly one above the one in registry
-    - change all occurrences in packages/tests/aea/examples/benchmark/docs to new reference
-    - change yaml version number
+    def __init__(
+        self, ask_version, update_version, replace_by_default, no_interactive, context
+    ):
+        """Init updater."""
+        self.option_ask_version = ask_version
+        self.option_update_version = update_version
+        self.option_replace_by_default = replace_by_default
+        self.option_no_interactive = no_interactive
+        self.option_context = context
+
+    @staticmethod
+    def check_if_svn_installed():
+        """Check svn tool installed."""
+        res = shutil.which("svn")
+        if res is None:
+            raise Exception("Install svn first!")
+
+    @staticmethod
+    def run_hashing():
+        """Run hashes update."""
+        hashing_call = update_hashes()
+        if hashing_call == 1:
+            raise Exception("Problem when running IPFS script!")
+
+    @staticmethod
+    def check_if_running_allowed():
+        """
+        Check if we can run the script.
+
+        Script should only be run on a clean branch.
+        """
+        git_call = subprocess.Popen(["git", "diff"], stdout=subprocess.PIPE)  # nosec
+        (stdout, _) = git_call.communicate()
+        git_call.wait()
+        if len(stdout) > 0:
+            raise Exception("Cannot run script in unclean git state.")
```

Comment:
These two are repeated above (lines 123-142). Remove one please.

Reply:
removed. forgot to remove after moving to the class